crypto-ffi/src/timestamp.rs

```diff
@@ -0,0 +1,45 @@
+//! Custom timestamp type for UniFFI that maps to kotlinx.datetime.Instant in Kotlin
+/// the reason behind it is to unify the timestamp mapping between the JVM/Android bindings
+/// and the new Kotlin Multiplatform bindings.
+/// this can be removed once we fully migrate to Kotlin Multiplatform and
+/// stop generating JVM/Android bindings.
+
+#[cfg(not(target_family = "wasm"))]
+use std::time::{Duration, SystemTime};
+
+#[cfg(not(target_family = "wasm"))]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct KotlinInstant(pub SystemTime);
```

Comment:
What does this do when built for iOS? (which to uniffi is also `cfg(not(target_family = "wasm"))`.)